### ISSUE TITLE
`command-instead-of-shell` should take into account the args of the `shell` module

### ISSUE THREAD

**Issue by nkakouros** [ansible/ansible-lint#2165]
##### Summary
The `command` module does not support since ansible 2.4 setting the executable via `args.executable`.  The `shell` module is needed then. But ansible-lint will complain that `command` was not used.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible [core 2.14.0.dev0]
  config file = None
  configured module search path = ['/home/nikos/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.10/site-packages/ansible
  ansible collection location = /home/nikos/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible
  python version = 3.10.4 (main, Mar 23 2022, 23:05:40) [GCC 11.2.0] (/usr/bin/python3.10)
  jinja version = 3.1.2
  libyaml = True

ansible-lint 6.2.1 using ansible 2.14.0.dev0
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

Ubuntu 18.04

##### STEPS TO REPRODUCE

```yaml
---
- hosts: localhost
  tasks:
    - shell: pwd
      args:
        executable: /bin/bash
```

##### Desired Behavior

I would expect ansible-lint to take the `executable` arg into account and not complain.

##### Actual Behavior

```console (paste below)
# among many other warnings

command-instead-of-shell: Use shell only when shell functionality is required.
playbook.yml:4 Task/Handler: shell executable=/bin/bash pwd

...
```

**Comment by ssbarnea** [comment]
@nkakouros Good catch! You are welcomed to make PR to fit this.
